pkg/util/util.go

```diff
@@ -185,3 +185,17 @@ func EnsureDirExists(dir string) error {
 
 	return err
 }
+
+func Backends() []string {
```

Comment:
I would prefer not to have this in the utils file as its another place to update. Can we have a function in the iai file?

Reply:
I will remove the function and have it as part of vars in iai file along with the ai clients.